drivers/dp/swdp_ll_pin.h

```diff
@@ -6,8 +6,9 @@
 
 #include <zephyr/kernel.h>
 #include <zephyr/drivers/gpio.h>
+#include <hal/nrf_gpio.h>
 
-#if defined(CONFIG_SOC_SERIES_NRF52X)
+#if defined(CONFIG_SOC_SERIES_NRF52X) || defined(CONFIG_SOC_SERIES_NRF53X)
```

Comment:
This looks like a separate logical change, related only to nRF53, so I'd prefer a distinct commit for that. Anyway, I agree this is needed to make SWDP driver work on nRF53 with default configuration.